utmap-client/src/components/EventList.js

```diff
@@ -0,0 +1,24 @@
+import React, { Component } from 'react';
+import Typography from '@material-ui/core/Typography';
+import PropTypes from 'prop-types';
+import EventListItem from './EventListItem';
+
+export class EventList extends Component {
```

Comment:
- Why do you need to export twice?
- I think we are using function for the components, but I would confirm first with Shahmeer before you change it.

Reply:
One is for default export, and one is for named exports, I believe. But it shouldn't matter since one was removed in conversion.
Also, converted to functional components.